### PR TITLE
[DOC] Improve documentation for TinyTimeMixer

### PR DESCRIPTION
**Pull request by geetu040**
Update the docstring for parameter `model_path` to more informative info.

### CODE REVIEW

File: sktime/forecasting/ttm.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ class TinyTimeMixerForecaster(_BaseGlobalForecaster):
     Parameters
     ----------
     model_path : str, default="ibm/TTM"
-        Path of the model to use for forecasting.
+        Path to the huggingface model to use for forecasting.
```
Comment (fkiraly) [inline sktime/forecasting/ttm.py:41]: Thanks! Still think this needs a bit more of reference context, i.e., is the path unique? It is only if you also know the huggingface repo or group, no?

Comment (geetu040) [reply sktime/forecasting/ttm.py:41]: path is the name of the hf repo actually, so you need to know the repo name to use here and you can also set the path to a local folder containing files supported by huggingface

Comment (geetu040) [reply sktime/forecasting/ttm.py:41]: can you explain what more needs to be added in the docstring, I cannot really think of any additional context to give here?

Comment (fkiraly) [reply sktime/forecasting/ttm.py:41]: * a path is always relative to a path root, or absolute, and only specified in a context ("what kind of path").
* a "huggingface model" is not a clearly specified artefact

The text you wrote could mean multiple things, including

* path relative to root of your primary hard drive to a huggingface weights file in xyz format
* URL path of the huggingface repository containing the model, where the repository is assumed to be of a special structure

**Review by fkiraly — APPROVED** [review]
Thanks, that´s what I meant. Could even be more precise, but this is a notable improvement already

**Pull request merged**